<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accounts_customer</class>
 <widget class="QWidget" name="accounts_customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #34495e;</string>
  </property>
  <widget class="QLineEdit" name="money_charg">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>330</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>برداشت</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>420</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>شماره حساب:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>شماره حساب و مبلغ را وارد کنید</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="charg">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d;  font-size:25px;</string>
   </property>
   <property name="text">
    <string>شارژ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="number_account_c">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>140</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>30</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>شارژ حساب</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>421</width>
     <height>491</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>419</width>
      <height>489</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="accounts">
       <property name="styleSheet">
        <string notr="true">color:white; font-size:18px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLineEdit" name="number_account_b">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>440</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bardasht">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d;  font-size:25px;</string>
   </property>
   <property name="text">
    <string>برداشت</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>190</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>مبلغ:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>490</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>مبلغ:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="subtract_money">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>510</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:20px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>411</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>شماره حساب و مبلغ را وارد کنید</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back_menu_pc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 24px;

	color:#EA2027; background-color:#bdc3c7; border: 2px solid #ff4d4d;  font-size:18px;</string>
   </property>
   <property name="text">
    <string>برگشت</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>120</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border: 0;
		text-align: center;
	background: #27ae60;
	display: block;
	margin: 3px 3px;
	text-align: center;
	border: 2px solid #2ecc71;
	outline: none;
	color: white;
	border-radius: 20px;
	font-size:20px;	
	text-align: center;</string>
   </property>
   <property name="text">
    <string>شماره حساب:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="inf_charg">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red; font-size:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="inf_bardasht">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>570</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red; font-size:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
